demo/index.js

```diff
@@ -59,8 +58,9 @@ function embedContent({
 	}));
 
 	function updateStatus() {
-		statusEl.innerHTML = currentEmbed.status || 'undefined';
-		playerStatusEl.innerHTML = currentEmbed.playerStatus || 'undefined';
+		statusEl.innerHTML = currentEmbed.isVod ?
```

Comment:
is this boolean inverted? if isVod === true then don't you want playerStatus?

Reply:
good point